Specs/Core/ArcGISTiledElevationTerrainProviderSpec.js

```diff
@@ -474,12 +474,11 @@ describe("Core/ArcGISTiledElevationTerrainProvider", function () {
         var promise;
         var i;
         for (i = 0; i < RequestScheduler.maximumRequestsPerServer; ++i) {
-          promise = terrainProvider.requestTileGeometry(
-            0,
-            0,
-            0,
-            createRequest()
-          );
+          var request = new Request({
+            throttle: true,
+            throttleByServer: true,
+          });
+          promise = terrainProvider.requestTileGeometry(0, 0, 0, request);
```

Comment:
Isn't it a problem that we're no longer calling `createRequest` here? So the request has no URL, for one thing.

Reply:
it seems like createRequest() returns an empty request object with only `throttleByServer` option on. I kind of create the new Request for only the test case. Should I modify the createRequest() with throttle to be true for all of them?

Reply:
Oh ok, I hadn't realized `createRequest` also created a request with a URL. That makes sense, no need to change anything.